deployment/dataproc/combine-and-clip.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This script and the other one below were ported directly from https://github.com/azavea/geospatial-apps/blob/master/gophillygo_deployment.md.